lightning-tag/user-consent.md

```diff
@@ -0,0 +1,126 @@
+# User Consent Guide
+
+## Common User Consent Use-Case
+
+For Lotame clients that want to provide their users the ability to opt-out of tracking, the Lotame Lightning Tag provides the ability pass the user's consent options. These consent options will be used during Lotame processing against that user's profile.
```

Comment:
For clients that utilize Lotame's Consent Management Product, consent signals can be passed directly into Lightning Tag. Use this feature to inform Lotame of each user's consent preference and how they should be handled within the DMP.

Reply:
👍